queries/julia/highlights.scm

```diff
@@ -5,13 +5,13 @@
 ((identifier) @constant
   (#match? @constant "^[A-Z][A-Z_]{2}[A-Z_]*$"))
 
-[
-  (triple_string)
-  (string)
-] @string
-(command_string) @string.special
+(character_literal) @string
```

Comment:
I think characters have a separate highlight

Reply:
Just tell me what to use and I'll change it.

Reply:
I think it's just `@character`. Can't check on mobile